pkg/arvo/neo/cod/std/src/fil/introduction.md

```diff
@@ -0,0 +1,36 @@
+# Chapter 0: Introduction
+
+Shrubbery is a new in-development application model for Urbit. This guide will introduce the foundational Shrubbery concepts and get you started writing shrubs of your own. *It is aimed at Urbit developers who have at least been through App Academy, or otherwise have at least written a Gall agent.*
+
+At time of writing, Shrubbery's build system lives in a Gall agent called `%neo`. You can install this agent on a comet or moon with `|install ~met %neo`. You can install this desk on a fakeship from the GitHub repo, as detailed below.
+
+## Cloning from GitHub
+1. Clone the `urbit/shrub` repo to your machine.
+2. Get the latest `urbit` [runtime](https://docs.urbit.org/manual/getting-started/self-hosted/cli).
+3. Boot a fakeship with the `--loom 33` flag.
+4. `|mount %base`
+5. `rsync -avL —delete urbit/shrub/pkg/arvo/* pier/base`
+6. `|commit %base`
+7. `|start %neo`
+
+This could take a while, you’ll see `booted %neo` in the Dojo when it’s complete.
+
+## Directory structure
+The only folders you need to understand for these tutorials are `/pro`, `/imp`, and `/con`.
+- `/pro` for protocols. Like `/sur`, this is where your type definitions live.
+- `/imp` for implementations. Like `/app`, this is where your app logic lives.
+- `/con` for conversions. Like `/mar`, this is where you define rules for transforming nouns in your desk.
```

Comment:
I'm torn on this comparison between con and mar, what do you think @tiller-tolbus ?